File: chapter_4/Csharp/Csharp/code2_private.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Csharp
{
    class person
    {
        public int intAge;
        private string strName;

        public void setName(string name)
        {
            strName = name;
        }

        public string getName()
        {
            return strName;
        }

        public void displayData()
        {
            Console.WriteLine("age= " + intAge);
            Console.WriteLine("name= " + strName + "\n");
        }
    }
    class code2_private
    {
        public static void display()
        {
            person pr1 = new person();
            pr1.intAge = 23;
            pr1.setName("Mahmoud");
            pr1.displayData();
        }
    }
}

File: chapter_4/vb/code_1/code2_private.vb
Module code2_private
    Class person
        Public intAge As Integer
        Private strName As String

        Public Sub setName(ByVal name)
            strName = name
        End Sub

        Public Function getName() As String
            Return strName
        End Function

        Public Sub displayData()
            Console.WriteLine("age= " & intAge)
            Console.WriteLine("name= " & strName & vbCrLf)
        End Sub
    End Class

    Sub Main()
        Dim objPerson As New person

        objPerson.intAge = 23
        objPerson.setName("Mahmoud")
        objPerson.displayData()

    End Sub

End Module
